### PR TITLE
refactor:replace joinAnd with htmlJoinAnd and replaced [conf.wgHTML] with just conf.wgHTML in src/core/w3c/sotd.js

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
refactor:replace joinAnd with htmlJoinAnd and replaced [conf.wgHTML] with just conf.wgHTML in src/core/w3c/sotd.js

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
Modification of same at
Screenshot from 2019-12-11 23-53-11
gives an error in tests at 
Screenshot from 2019-12-11 23-52-37
Screenshot from 2019-12-11 23-57-55

saying el  is null.
I feel that tests need some modification because hyperhtml is used and hence no element is filtered.
If I am correct, shall I go on to do the same
@saschanaz Please help me in this

**Comment by SuyashSalampuria** [comment]
@saschanaz @sidvishnoi  @marcoscaceres  Please review this

**Comment by SuyashSalampuria** [comment]
@saschanaz please review this

**Comment by SuyashSalampuria** [comment]
@saschanaz @marcoscaceres please review this

File: src/core/issues-notes.js [hunk 1]
```diff
@@ -266,12 +266,16 @@ function isLight(rgb) {
 function createLabelsGroup(labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
   const labelNames = labels.map(label => label.name);
-  const joinedNames = joinAnd(labelNames);
+  const joinedNames = document.createElement("div");
```
Comment (saschanaz) [inline src/core/issues-notes.js]: This specific one looks okay as it doesn't contain any HTML and is not inserted through `${[...]}`. It's safe as-is.

File: src/w3c/headers.js [hunk 2]
```diff
@@ -537,15 +537,14 @@ export function run(conf) {
     const pats = [];
     for (let i = 0, n = conf.wg.length; i < n; i++) {
       pats.push(
-        `a <a href='${conf.wgPatentURI[i]}' rel='disclosure'>` +
-          `public list of any patent disclosures (${conf.wg[i]})</a>`
+        hyperHTML`a <a href='${conf.wgPatentURI[i]}' rel='disclosure'>public list of any patent disclosures (${conf.wg[i]})</a>`
       );
     }
-    conf.wgPatentHTML = joinAnd(pats);
+    conf.wgPatentHTML = htmlJoinAnd(pats);
```
Comment (saschanaz) [inline src/w3c/headers.js]: Could you also replace `${[conf.wgPatentHTML]}` into `${conf.wgPatentHTML}` in `src/w3c/templates/sotd.js`?

File: src/w3c/headers.js [hunk 3]
```diff
@@ -433,11 +433,11 @@ export function run(conf) {
...
   if (conf.bugTracker) {
     if (conf.bugTracker.new && conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
     } else if (conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.open}'>open bugs</a>`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.open}'>open bugs</a>`;
     } else if (conf.bugTracker.new) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>file a bug</a>`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.new}'>file a bug</a>`;
```
Comment (saschanaz) [inline src/w3c/headers.js]: Same here, please replace `${[conf.bugTrackerHTML]}` into `${conf.bugTrackerHTML}` in two `src/w3c/templates/(cgbg-)headers.js` files.

**Comment by saschanaz** [comment]
>Modification of same at .. gives an error in tests at .. saying el is null.

I would prefer doing that in another PR. That one specifically needs the solution I mentioned in https://github.com/w3c/respec/pull/2597#discussion_r353717409.

BTW, sorry for late response!

**Pull request closed without merging**